client/src/App.js

```diff
@@ -20,32 +18,10 @@ import './styles/styles.css';
  */
  const App = () => {
  return (
-  <div className="app-container">
-    <Navbar />
-
-    <div className="content-container">
-      <Sidebar>
-        <Menu>
-          <SubMenu label="Patients">
-            <MenuItem> Create and View Patients </MenuItem>
-            <MenuItem> Medication Info </MenuItem>
-          </SubMenu>
-          <MenuItem> Calculate </MenuItem>
-          <MenuItem> Trends </MenuItem>
-          <MenuItem> Settings </MenuItem>
-          <MenuItem> Log Out </MenuItem>
-        </Menu>
-      </Sidebar>
-
-      <div className="main-content">
-        <Routes>
-          <Route exact path="/" element={<RecordList />} />
-          <Route path="/edit/:id" element={<Edit />} />
-          <Route path="/create" element={<Create />} />
-        </Routes>
-      </div>
-    </div>
-  </div>
- );
+  <Routes>
```

Comment:
Why route to calculator here, but not patients? Is this the default page? If so, that works!

Reply:
The default page later on will be the login page.